dom-manipulation/main.js

```diff
@@ -0,0 +1,29 @@
+const $hotButton = document.querySelector('.hot-button');
+const $clickCount = document.querySelector('.click-count');
+let $counter = 0;
```

Comment:
No `$` here! We only include it in front of variable names that are storing DOM elements.